include/exadg/grid/grid.cpp

```diff
@@ -144,6 +142,8 @@ Grid<dim>::create_triangulation(
     // coarse grid triangulations
     if(data.triangulation_type == TriangulationType::Serial)
     {
+      AssertThrow(triangulation->all_reference_cells_are_hyper_cube(), dealii::ExcNotImplemented());
```

Comment:
In my opinion, this needs some more explanations. As a new user/developer of ExaDG, I can deduce that Serial requires hypercubes, but I would like to know the reason, e.g. that this is a current limitation of deal.II. Why does simplex not work here? I think this should be mentioned here. Imagine deal.II becomes more powerful. How should a developer of ExaDG know in 1-2 years why we have this assert in the code and how should he decide whether it can be removed or not. A developer might remove it and then realize it works, but he will then ask himself whether this is just luck and whether it will work in all cases ...

Reply:
That is why I used `dealii::ExcNotImplemented()` and not `ExaDG::ExcNotImplemented()`. I thought this made clear, that currently this is not allowed, because of dealii and not because of ExaDG. However, I agree that adding more explanation would be helpful.

Reply:
changed in fe49537a6dd436ad6b49130feb488cd584e5d308.